pages.de/common/deno.md

```diff
@@ -0,0 +1,24 @@
+# deno
+
+> Eine sichere JavaScript- und TypeScript-Laufzeit.
+> Weitere Informationen: <https://deno.land/>.
+
+- Führe eine JavaScript oder TypeScript Datei aus:
+
+`deno run {{pfad/zu/datei.ts}}`
+
+- Starte ein REPL (interaktive Konsole):
+
+`deno`
+
+- Führe eine Datei mit Netzwerkzugriff aus:
```

Comment:
I checked this command description with Google translate ( and it translated to `Run a file with network access:`) shouldn't it be `Führen Sie eine Datei mit aktiviertem Netzwerkzugriff aus:` (Run a file with network access enabled:). This is just a small suggestion, feel free to mark this conversation as resolved if this isn't necessary.

```suggestion
- Führen Sie eine Datei mit aktiviertem Netzwerkzugriff aus:
```

Reply:
`Führen Sie` is quite a formal version, whereas the rest of the german translations I found used the imperative version.
`Netzwerkzugriff` vs. `aktiviertem Netzwerkzugriff` is basically `network access` vs. `activated network access`, where I preferred the version in my PR.

Reply:
Thanks for the insights 😄 , I think your current version is fine.